src/components/Tags/Tag.utils.ts

```diff
@@ -1,42 +1,50 @@
 import { Tag } from "../../features/entity.types";
 
-export const defaultTagsList: string[] = [
+export const DefaultTagsList: string[] = [
```

Comment:
the DefaultTagsList would be a really good one to convert to an enum if you're not familiar with them